dependencies/clients/shared.go

```diff
@@ -315,6 +313,14 @@ func (client *clientBinary) IsRunning() bool {
 }
 
 func initClient(ctx *cli.Context, client ClientBinaryDependency) (err error) {
+	if utils.FileExists(ctx.String(flags.GethDatadirFlag)) { // geth datadir is the same as erigon - no matter which client we use
+		log.Info("⚙️  Database already exists - continuing...")
```

Comment:
We should say: `Execution database` here.

Reply:
Done